documentation/modules/deploying/proc-deploy-migrate-kraft.adoc

```diff
@@ -209,17 +209,24 @@ NOTE: Depending on how `deleteClaim` is configured for ZooKeeper, its Persistent
 
 . Remove any ZooKeeper-related configuration from the `Kafka` resource.
 +
-If present, you can remove the following:
+Remove the following section:
 +
+--
+* `spec.zookeeper` properties
```

Comment:
It should be just `property` as singular or I guess you can remove the word `properties` because you are saying "section" before.